style.css

```diff
@@ -1,7 +1,55 @@
+body {
+  display: flex;
+  flex-direction: column;
+  border: #1a2236 solid;
```

Comment:
- [OPTIONAL] I think you forgot to remove this border. I think it is just for styling sketching 😅

Reply:
It's true, was for styling sketching only, Thank you